contrib/check-config.sh

```diff
@@ -181,6 +181,9 @@ echo
 
 echo 'Optional Features:'
 {
+	check_flags USER_NS
+}
+{
```

Comment:
@runcom  I just want make this more readable since `USER_NS` is namespace config and those are cgroup config, so I make this a new block.